sdk/identity/identity/README.md

```diff
@@ -37,7 +37,7 @@ To authenticate in Visual Studio Code, first ensure the [Azure Account Extension
 
 Applications using the `AzureCliCredential`, rather directly or via the `DefaultAzureCredential`, can use the Azure CLI account to authenticate calls in the application when running locally.
 
-To authenticate with the [Azure CLI][azure_cli] users can run the command `az login`. For users running on a system with a default web browser the azure cli will launch the browser to authenticate the user.
+To authenticate with the [Azure CLI][azure_cli] users can run the command `az login`. For users running on a system with a default web browser the Azure cli will launch the browser to authenticate the user.
```

Comment:
Hello! Thank you for making this pull request. I appreciate the effort you're putting here! I like these casing on the Azure name, very necessary! Thanks!